### PR TITLE
[GAL-709] fix bio appears weird in the follower list modal

### PR DESCRIPTION
**Pull request by jakzaizzat**
**Changes**
Remove all the breakline for bio **only** in follower list modal

**Before**
CleanShot 2022-11-23 at 10 06 39

**After**
CleanShot 2022-11-23 at 10 04 19

### CODE REVIEW

File: src/components/Follow/FollowListUsers.tsx [hunk 1]
```diff
@@ -35,12 +36,16 @@ export default function FollowListUsers({
     track('Follower List Username Click');
   }, [track]);
 
+  const trimBreaklines = useCallback((text: string) => {
+    return text.replace(BREAK_LINES, '');
+  }, []);
```
Comment (Robinnnnn) [inline src/components/Follow/FollowListUsers.tsx:41]: we should just be able to useMemo here!

```js
const bio = useMemo(() => user.bio.replace(BREAK_LINES, ''), [user.bio])
```

File: src/utils/regex.ts [hunk 2]
```diff
@@ -10,3 +10,6 @@ export const VALID_URL =
 
 // standard email format x@x.xx
 export const EMAIL_FORMAT = /^[\w-\.]+@([\w-]+\.)+[\w-]{2,4}$/;
+
+// break lines
+export const BREAK_LINES = /(\r\n|\n|\r|\\\n)/gm;
```
Comment (Robinnnnn) [inline src/utils/regex.ts:15]: nit: would love to see some unit tests with this

File: src/components/Follow/FollowListUsers.tsx (hunk 1 above)
Comment (jakzaizzat) [reply src/components/Follow/FollowListUsers.tsx:41]: fml. why did I used `useCallback` here 🤦🏻‍♂️

Comment (jakzaizzat) [reply src/components/Follow/FollowListUsers.tsx]: Updated!

File: src/utils/regex.ts (hunk 2 above)
Comment (jakzaizzat) [reply src/utils/regex.ts:15]: Added unit test for this regex!

**Review by Robinnnnn — APPROVED** [review]
LFG

**Pull request merged**